src/pages/ProductPage/ProductFilters.jsx

```diff
@@ -0,0 +1,148 @@
+
+const ProductFilters = () => {
+const { state, dispatch } = useProduct();
+const { sortBy, categories, price } = state;
+const {food, leash, toys, clothes, small, medium, large, arrival, choice, bestseller} = categories;
+
+return (
+<aside className="filters-sec p-2" style={{height: '85vh'}}>
+    <div className="filter-con flex-row-sb my-sm">
+        <button>
+            <h3>FILTERS</h3>
+        </button>
+
+        <a className="link" href="#">
+            <h5><button className="clear-all-btn" onClick={() => dispatch({type: "CLEAR"})}>clear all</button></h5>
+        </a>
+    </div>
+    <hr />
+    <div className="filter-content">
+        <div className="sort-con my-sm">
+            <h5>SORT BY</h5>
+            <form action="#" className="m-0 py-1 flex-col">
+
+                <label htmlFor="sort-h-t-l">
+                    <input type="radio" id="sort-h-t-l" name="sort" className="input" checked={sortBy==="HIGH_TO_LOW" }
+                        onChange={()=> dispatch({ type: "HIGH_TO_LOW" })}
+                    />
+                    Price: High to Low
+                </label>
+
+                <label htmlFor="sort-l-t-h">
+                    <input type="radio" id="sort-l-t-h" name="sort" className="input" checked={sortBy==="LOW_TO_HIGH" }
+                        onChange={()=> dispatch({ type: "LOW_TO_HIGH" })}
+                    />
+                    Price: Low to High
+                </label>
+
+                <label htmlFor="sort-l-t-h">
+                    <input type="radio" id="sort-l-t-h" name="sort" className="input" checked={sortBy==="RATING_HIGH_TO_LOW" }
+                        onChange={()=> dispatch({ type: "RATING_HIGH_TO_LOW" })}
+                    />
+                    Rating: High to Low
+                </label>
+
+                <label htmlFor="sort-l-t-h">
+                    <input type="radio" id="sort-l-t-h" name="sort" className="input" checked={sortBy==="RATING_LOW_TO_HIGH" }
+                        onChange={()=> dispatch({ type: "RATING_LOW_TO_HIGH" })}
+                    />
+                    Rating: Low to High
+                </label>
+            </form>
+        </div>
+
+        <div className="price-range my-md">
+            <h5>PRICE</h5>
+            <label htmlFor="price-range">
+                <input id="price-range" type="range" step="100" min="100" max="500" value={price} onChange={(e)=>
+                dispatch({ type: "PRICE", price: e.target.value })}
+                />
+                {price} and below
+            </label>
+        </div>
+
+        <div className="sort-con">
+            <h5 className="py-sm">CATEGORIES</h5>
+            <form name="categories" className="categories flex-col">
+                <h6 className="p-sm">Products</h6>
+                <label htmlFor="food">
+                    <input type="checkbox" className="input" id="food" value="food" name="categories-name" checked={
```

Comment:
Instead of specifying every category specifically or hard coding every category, you should take these values from category array that you get from backend.
This will make your code extensible and you will be saved from doing extra "not-required" typing.